msal/tests/dev apps/WebTestApp/WebApp/Startup.cs

```diff
@@ -98,7 +98,7 @@ public void ConfigureServices(IServiceCollection services)
         public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory)
         {
             // Add the console logger.
-            loggerFactory.AddConsole(LogLevel.Debug);
+            loggerFactory.AddConsole(Microsoft.Extensions.Logging.LogLevel.Debug);
```

Comment:
Is this namespace expected?

Reply:
Ah I see, this is so that you differentiate between our logger and the asp.net logger. Cool